File: backend/apps/lldp/lldp.py
class Lldp():
    def __init__(self, iface_name, neighbor_id, neighbor_first_message, neighbor_last_update, neighbor_name, neighbor_iface):
        self.iface_name = iface_name
        self.neighbor_id = neighbor_id
        self.neighbor_first_message = neighbor_first_message
        self.neighbor_last_update = neighbor_last_update
        self.neighbor_name = neighbor_name
        self.neighbor_iface = neighbor_iface

File: frontend/src/models/Lldp.ts
import { ISOtoString } from "@/services/ConvertDate";

export default class Lldp {
    iface_name!: string;
    neighbor_id!: string;
    neighbor_first_message!: string;
    neighbor_last_update!: string;
    neighbor_name!: string;
    neighbor_iface!: string;

    constructor(jsonObj?: Lldp) {
        if (jsonObj) {
            this.iface_name = jsonObj.iface_name;
            this.neighbor_id = jsonObj.neighbor_id;
            this.neighbor_first_message = jsonObj.neighbor_first_message;
            this.neighbor_last_update = jsonObj.neighbor_last_update;
            this.neighbor_name = jsonObj.neighbor_name;
            this.neighbor_iface = jsonObj.neighbor_iface;

            if (jsonObj.neighbor_first_message) {
                this.neighbor_first_message = ISOtoString(this.neighbor_first_message);
            }
            if (jsonObj.neighbor_last_update) {
                this.neighbor_last_update = ISOtoString(this.neighbor_last_update);
            }
        }

    }


}